src/compiler/parse/state/tag.ts

```diff
@@ -221,9 +221,9 @@ export default function tag(parser: Parser) {
 		element.children = read_sequence(
 			parser,
 			() =>
-				parser.template.slice(parser.index, parser.index + 11) === '</textarea>'
+				/<\/textarea[>\s]/i.test(parser.template.slice(parser.index, parser.index + 11))
```

Comment:
Is this safe? It seems like you should be ensuring at this step that there is a `>` after the whitespace. What happens if the `.read` call below fails?

Reply:
ooh you are right!

reading deeper i realise that it doesn't have to be just whitespace, as long as there's 1 whitespace, anything character until `>` are considered as attributes to the ending tag

see: https://codepen.io/tanhauhau/pen/BaWerro